### ISSUE TITLE
Invalid Int32

### ISSUE THREAD

**Issue by afinetooth** [coverallsapp/coverage-reporter#5]
(Reporting for client.)

# Issue

This command:

```
wget -cq https://coveralls.io/coveralls-linux.tar.gz -O - | tar -xz && ./coveralls --repo-token=<REMOVED> -flcov.info --debug
```

Results in this error:

> Invalid Int32: 3028760000

![invalid-int32_universal_reporter](https://user-images.githubusercontent.com/918667/146990974-f666d5f9-2895-406d-8f1b-7b2e684b738e.png)

## Supporting info

- https://crystal-lang.org/api/1.2.2/Int32.html
- Don't have it, but the lcov file would help. Will append if possible.

**Comment by afinetooth** [comment]
Here is the LCOV file (~10MB):

(Added `.txt` to filename `lcov.info` to allow for upload here.)

[lcov.info.txt](https://github.com/coverallsapp/coverage-reporter/files/7770309/lcov.info.txt)

**Comment by mrexox** [comment]
This issue was fixed in 0.2.0 version. Closing it.
